db/models/articlesModel.js

```diff
@@ -0,0 +1,20 @@
+const db = require('../connection');
+
+exports.selectAllArticles = (req) => {
+    const articleId =  Object.values(req.params);
```

Comment:
I would move line 4 into your controller, theres no need to send the whole request object into this function